test/mongooseim_metrics_SUITE.erl

```diff
@@ -91,6 +105,15 @@ tcp_connections_detected(_C) ->
     {ok, [{value, X}]} = mongoose_metrics:get_metric_value(global, tcpPortsUsed),
     ?assert(X > 0).
 
+queued_messages_increase(_C) ->
+    Pids = [spawn(fun() -> receive die -> ok end end) || _ <- lists:seq(1,5)],
+    lists:foreach(fun(Pid) -> Pid ! undefined end, Pids),
+    timer:sleep(60),
```

Comment:
Instead of single sleep, let's wrap the verification in `async_helper:wait_until/2`.